lib/adyen/rest/errors.rb

```diff
@@ -11,23 +11,38 @@ class RequestValidationFailed < Adyen::REST::Error
 
     # Exception class for error responses from the Adyen API.
     #
-    # @!attribute category
+    # @!attribute status
     #    @return [String, nil]
-    # @!attribute code
+    # @!attribute error_code
     #    @return [Integer, nil]
-    # @!attribute description
+    # @!attribute message
+    #    @return [String, nil]
+    # @!attribute error_type
+    #    @return [String, nil]
+    # @!attribute psp_reference
     #    @return [String, nil]
     class ResponseError < Adyen::REST::Error
```

Comment:
Could also be implemented with the same attribute access pattern as in the Response object (thru response[:parameter_name])